src/app/beer_garden/systems.py

```diff
@@ -280,3 +343,15 @@ def handle_event(event: Event) -> None:
 
         elif event.name == Events.SYSTEM_REMOVED.name:
             db.delete(event.payload)
+
+        elif event.name == Events.INSTANCE_UPDATED.name:
+            if not event.payload_type:
+                logger.error(f"{event.name} error: no payload type ({event!r})")
+                return
+
+            record = db.query_unique(Instance, id=event.payload.id)
+
```

Comment:
Why are you not using something like `_from_kwargs` function that you made in `plugin.py` to pull back the instance record OR the `get_instance` function above?

Reply:
This is just another one I missed. I'll need to tweak how this works.